src/actions/photos.js

```diff
@@ -0,0 +1,106 @@
+/**
+  Photos related features
+**/
+
+import cozy from 'cozy-client-js'
+
+import {
+  COZY_PHOTOS_DIR_ID,
+  FETCH_PHOTOS,
+  FETCH_PHOTOS_FAILURE,
+  RECEIVE_PHOTOS,
+  UPLOAD_PHOTOS,
+  UPLOAD_PHOTOS_SUCCESS,
+  UPLOAD_PHOTOS_SUCCESS_WITH_CONFLICTS,
+  UPLOAD_PHOTOS_FAILURE
+} from './constants'
```

Comment:
I've already said this but I am not a fan at all of a using a dedicated constants files. In my opinion, every constant should rely on a given context and be stored in the associated files. What's your opinion @goldoraf ?